docs/source/tutorials/linux/dvkm/results.md

```diff
@@ -0,0 +1,597 @@
+   0xffffffffc0000c4d <+29>:    ja     0xffffffffc0000c91 <dvkm_ioctl+97>
+   0xffffffffc0000c4f <+31>:    cmp    $0xc0184402,%esi
+   0xffffffffc0000c55 <+37>:    je     0xffffffffc0000e18 <dvkm_ioctl+488>
+   0xffffffffc0000c5b <+43>:    jbe    0xffffffffc0000d1e <dvkm_ioctl+238>
+   0xffffffffc0000c61 <+49>:    cmp    $0xc0184403,%esi
+   0xffffffffc0000c67 <+55>:    je     0xffffffffc0000d9f <dvkm_ioctl+367>
+   0xffffffffc0000c6d <+61>:    cmp    $0xc0184405,%esi
+   0xffffffffc0000c73 <+67>:    jne    0xffffffffc0000c89 <dvkm_ioctl+89>
+   0xffffffffc0000c75 <+69>:    mov    $0xffffffffc00104a0,%rdi
+   0xffffffffc0000c7c <+76>:    call   0xffffffff8127e7a0 <_printk>
+   0xffffffffc0000c81 <+81>:    mov    %rbp,%rdi
+   0xffffffffc0000c84 <+84>:    call   0xffffffffc0000a20 <Stack_OOBR_IOCTL_Handler>
+   0xffffffffc0000c89 <+89>:    xor    %eax,%eax
+   0xffffffffc0000c8b <+91>:    pop    %rbp
+   0xffffffffc0000c8c <+92>:    ret
+:::
+
+To continue playing with our breakpoints, let's move to `oops_enter` (confirming that the current payload is triggering a crash), and dump the stack:
+
+:::{code-block} shell
+---
+caption: Dumping the stack on oops_enter
+---
+Breakpoint 2, oops_enter () at kernel/panic.c:623
+623     {
+(gdb) c
+(gdb) bt
+#0  oops_enter () at kernel/panic.c:623
+#1  0xffffffff810812dc in oops_begin () at arch/x86/kernel/dumpstack.c:338
+#2  0xffffffff81081cfe in die_addr (str=str@entry=0xffff888007f27b2c "general protection fault, probably for non-canonical address 0xe0000be0d732a202", regs=regs@entry=0xffff888007f27bb8, err=err@entry=0,
+    gp_addr=-2305829948902694398) at arch/x86/kernel/dumpstack.c:454
+#3  0xffffffff83c43378 in __exc_general_protection (error_code=0, regs=0xffff888007f27bb8) at arch/x86/kernel/traps.c:784
+#4  exc_general_protection (regs=0xffff888007f27bb8, error_code=0) at arch/x86/kernel/traps.c:729
+#5  0xffffffff83e01206 in asm_exc_general_protection () at ./arch/x86/include/asm/idtentry.h:564
+#6  0xffff888007f27e80 in ?? ()
+#7  0xffffffff86060000 in hprintf_buffer ()
+#8  0x1ffff11000fe4f90 in ?? ()
+#9  0xffffffff8605f044 in hprintf_buffer ()
+#10 0xffffffff8605f012 in hprintf_buffer ()
+#11 0x00007f06b9951016 in ?? ()
+#12 0x1ffffffff092a03d in ?? ()
+#13 0xffffffffc0010175 in ?? ()
+#14 0x203a61746164205d in ?? ()
+#15 0xdffffc0000000000 in ?? ()
+#16 0x00000fe0d732a202 in ?? ()
+#17 0x00320a00ffffff04 in ?? ()
+#18 0x0000000000000005 in fixed_percpu_data ()
+#19 0x0000000000000000 in ?? ()
+:::
+
+:::{Important}
+We need to use GDB hardware breakpoints, as software breakpoints in QEMU's embedded GDB server seems unrealiable.
+
+::::{Note}
+To reach the `dvkm_ioctl` hardware breakpoint, we had to send a `CTRL-C` to the GDB client.
+The execution was hanging somewhere, for reasons that are not clear at the time of this writing.
+::::
+::::{Note}
+Also remember that in order to load the symbols reliably, at the same location, we had to disable the Kernel `ASLR`.
```

Comment:
Remove the "also". This makes you referring to a previous statement, but since you have each note very visibly in a separate box the "Also" stands on its own and I have to search the previous text or boxes for something this relates to...?